styles/_variables.scss

```diff
@@ -1,10 +1,9 @@
 /* Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
 SPDX-License-Identifier: Apache-2.0 */
 
-
 $page-width-small-threshold: 850px;
 $page-width-medium-threshold: 1175px;
+$row-width-full-threshold: 1050px;
```

Comment:
this variable name is ambiguous and it is not obvious what it is being applied to.